### PR TITLE
Latest generator

### PR DESCRIPTION
**Pull request by ahuang11**
## Summary

Use latest template and generator to auto-generate this.

## Checklist
- [x] Summarized PR's changes in [CHANGELOG.md](https://github.com/PrefectHQ/prefect-databricks/blob/main/CHANGELOG.md)
- [x] Add example on converting JSON in README
- [x] Do an integration test
- [x] Fix missing models in docs?

### CODE REVIEW

**Review by desertaxle — CHANGES_REQUESTED** [review]

File: docs/models.md [hunk 1]
```diff
@@ -0,0 +1 @@
+::: prefect_databricks.models
```
[the remark this answers is not in the document]
Comment (ahuang11) [reply docs/models.md:1]: It seems to work for me.
image

[the remark this answers is not in the document]
Comment (desertaxle) [reply docs/models.md:1]: Interesting. Should be good then. We'll just make sure to check that it works when deployed to GitHub pages.

**Comment by ahuang11** [comment]
Not too sure how to handle missing models in docstring. The root problem is that the models don't have docstrings, so mkdocs hides them.

For example, the first model:
```
class AutoScale(BaseModel):
    class Config:
        extra = Extra.allow
        allow_mutation = False

    max_workers: Optional[int] = Field(
        None,
        description=(
            "The maximum number of workers to which the cluster can scale up when"
            " overloaded. max_workers must be strictly greater than min_workers."
        ),
    )
    min_workers: Optional[int] = Field(
        None,
        description=(
            "The minimum number of workers to which the cluster can scale down when"
            " underutilized. It is also the initial number of workers the cluster has"
            " after creation."
        ),
    )
```

This doesn't show in the docs because it's missing a docstring:
image

However, if I manually input a dummy docstring:
```
class AutoScale(BaseModel):
    """a"""
```
image

It shows up, but it doesn't include args.

**Comment by ahuang11** [comment]
I couldn't find a way to monkeypatch datamodel code generator, so I did a simple string replacement instead. Seems to work well:

image

**Review by desertaxle — CHANGES_REQUESTED** [review]
I think this is good to go once we sort out a couple of items around the `DatabricksCredentials` block.

File: README.md [hunk 2]
```diff
@@ -22,24 +26,104 @@ Install `prefect-databricks` with `pip`:
...
+from prefect_databricks.jobs import jobs_runs_submit
+from prefect_databricks.models.jobs import (
+    AutoScale,
+    AwsAttributes,
+    JobTaskSettings,
+    NotebookTask,
+    NewCluster,
+)
+
+DATABRICKS_INSTANCE = "dbc-ab1c23d4-567e.cloud.databricks.com"
```
[the remark this answers is not in the document]
Comment (ahuang11) [reply README.md]: Okay, I've refactored the generator to automatically parse the necessary arguments in the server base_url

**Review by desertaxle — APPROVED** [review]

**Review by desertaxle — APPROVED** [review]

**Pull request merged**